HelpHunterBE/Program.cs

```diff
@@ -29,23 +29,23 @@
 builder.Services.AddAuthentication(JwtBearerDefaults.AuthenticationScheme)
     .AddJwtBearer(options =>
     {
-        string signingKey = Environment.GetEnvironmentVariable("JWT_KEY");
-
-        if (string.IsNullOrEmpty(signingKey))
-        {
-            throw new ApplicationException("JWT_KEY not found in environment variables.");
-        }
-
-        options.TokenValidationParameters = new TokenValidationParameters
-        {
-            ValidateIssuer = true,
-            ValidateAudience = true,
-            ValidateLifetime = true,
-            ValidateIssuerSigningKey = true,
-            ValidIssuer = jwtConfig.Issuer,
-            ValidAudience = jwtConfig.Audience,
-            IssuerSigningKey = new SymmetricSecurityKey(Encoding.UTF8.GetBytes(signingKey))
-        };
+        //string signingKey = Environment.GetEnvironmentVariable("JWT_KEY");
```

Comment:
dlaczego to jest wykomentowane? Czy to przypadkiem nie zostało z testowania na lokalu?